pocs/tests/test_rs232.py

```diff
@@ -56,6 +56,11 @@ def test_detect_bogus_scheme(handler):
     assert '# bogus #' in repr(excinfo.value)
 
 
+def test_custom_logger(handler, fake_logger):
+    s0 = rs232.SerialData(port='no_op://', logger=fake_logger)
+    s0.logger.debug('Testing logger')
```

Comment:
What validation could we have here?

Reply:
I was a bit stumped on that. We could write different messages to different log files I suppose. I'll see if I can add something.